src/MapFactoryImpl1.cpp

```diff
@@ -41,9 +42,9 @@ static auto reg_host_phys_acc_splus = mpart::MapFactory::CompFactoryImpl<Kokkos:
 REGISTER_MONO_COMP(BasisHomogeneity::Homogeneous, PhysicistHermite, Exp, AdaptiveClenshawCurtis, Kokkos::HostSpace)
 REGISTER_MONO_COMP(BasisHomogeneity::Homogeneous, PhysicistHermite, SoftPlus, AdaptiveClenshawCurtis, Kokkos::HostSpace)
 #if defined(MPART_ENABLE_GPU)
-REGISTER_MONO_COMP(BasisHomogeneity::Homogeneous, PhysicistHermite, Exp, AdaptiveClenshawCurtis, mpart::DeviceSpace)
-REGISTER_MONO_COMP(BasisHomogeneity::Homogeneous, PhysicistHermite, Softplus, AdaptiveClenshawCurtis, mpart::DeviceSpace)
+//REGISTER_MONO_COMP(BasisHomogeneity::Homogeneous, PhysicistHermite, Exp, AdaptiveClenshawCurtis, mpart::DeviceSpace)
```

Comment:
I'm going to suggest we keep these in; just configure with `-DMPART_ARCHIVE=OFF` for the time being and then we'll fix them once tackling serialization.